public_dropin_gpu_environments/vllm/start_server.sh

```diff
@@ -14,4 +14,6 @@ echo "GPU count: $GPU_COUNT"
 echo
 echo "Starting DRUM server..."
 echo
+
+. ${DATAROBOT_VENV_PATH}/bin/activate
```

Comment:
I don't really love this... it means stuff in custom models behaves differently than functional tests (because they don't run this script). It's better to setup PATH in Dockerfile (which is all that activate script does anyway)

Reply:
But going forward DRUM at the very least shouldn't really on VIRTUAL_ENV variable.

Reply:
OK, activate wont do in Dockerfile what I'd want. Well. I can set both freaking variables then :)